packages/viewer/src/modules/LegacyViewer.ts

```diff
@@ -180,6 +180,10 @@ export class LegacyViewer extends Viewer {
     this.sections.enabled = true
   }
 
+  public setSectionBoxVisibility(visible: boolean) {
```

Comment:
@AlexandruPopovici I made this change to test my frontend work, it works so I've left it in, but can you check incase I've done something wrong here?

One problem is that when the frontend updates the section box visibility, I need to slightly move the camera for the change to be visible. EG. Turn off section box visibility. Section box will remain, until I slightly move the camera. Is this something that should be fixed in the viewer? I can probably fix in frontend, but it would be a bit hacky. 

Don't think there's any great urgency in this task.

Reply:
@andrewwallacespeckle 
The viewer employs on-demand rendering. This means that it is not constantly rendering, but rather it only does so when it needs to, or when it's told to. We're doing this mostly for mobile platform's sake where rendering each frame would drain battery pointlessly.
That's why whenever the client does something that needs to trigger a render, it needs to call `requestRender`, otherwise no change can be seen since there is no rendering happening. Moving the camera triggers a render request implicitly, that's why you were seeing that behavior.

I've added the calls to `requestRender` in the places that were missing it. Also, I've added a better approach on showing/hiding the section box that does not go through the `LegacyViewer`. We shouldn't add functionality in there, we actually need to retire it.

Let me know if you're having other issues with this

Reply:
@AlexandruPopovici, that's working perfectly! Thanks for the explanation too, I'll know better for next time.